src/ScreenHeader/HeaderButton.tsx

```diff
@@ -1,21 +1,78 @@
 import {AccessibilityProps, TouchableOpacity} from 'react-native';
 import React from 'react';
 import insets from '../utils/insets';
+import {LeftButtonProps, RightButtonProps} from './index';
+import {useNavigation} from '@react-navigation/core';
```

Comment:
Always import from `'@react-navigation/native'`, `core` should not be used directly